pandas/tests/dtypes/test_inference.py

```diff
@@ -67,17 +67,24 @@ def __getitem__(self):
     "ll",
     [
         [], [1], (1, ), (1, 2), {'a': 1},
-        {1, 'a'}, Series([1]),
+        Series([1]),
         Series([]), Series(['a']).str,
-        np.array([2])])
+        np.array([2])
+    ])
 def test_is_list_like_passes(ll):
-    assert inference.is_list_like(ll)
+    assert inference.is_list_like(ll, strict=False)
+
+
+def test_is_list_like_strict():
+    ll = {1, 'a'}
+    assert inference.is_list_like(ll, strict=False)
+    assert not inference.is_list_like(ll, strict=True)
```

Comment:
test should cover the `strict=None` case: verify that the warning is raised, and the behavior is consistent with `strict=False`

Reply:
also add issue number